testing/integration/logs_ingestion_test.go

```diff
@@ -110,16 +109,56 @@ func TestLogIngestionFleetManaged(t *testing.T) {
 }
 
 func startMockES(t *testing.T) string {
-	registry := metrics.NewRegistry()
 	uid := uuid.Must(uuid.NewV4())
 	clusterUUID := uuid.Must(uuid.NewV4()).String()
 
 	mux := http.NewServeMux()
-	mux.Handle("/", mockes.NewAPIHandler(
-		uid,
-		clusterUUID,
-		registry,
-		time.Now().Add(time.Hour), 0, 0, 0, 100, 0))
+	mux.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("X-Elastic-Product", "Elasticsearch")
+		w.Header().Set(http.CanonicalHeaderKey("Content-Type"), "application/json")
+		switch {
+		case r.Method == http.MethodGet && r.URL.Path == "/":
+			root := fmt.Sprintf("{\"name\" : \"mock\", \"cluster_uuid\" : \"%s\", \"version\" : { \"number\" : \"0.0.0\", \"build_flavor\" : \"default\"}}", clusterUUID)
```

Comment:
These raw JSON test templates are very ugly. Couldn't we use a map and just serialize it to JSON?